chain/wasm/src/plutus/utils.rs

```diff
@@ -0,0 +1,128 @@
+use cml_chain::plutus::Language;
+use wasm_bindgen::prelude::{wasm_bindgen, JsError, JsValue};
+use cml_crypto_wasm::ScriptHash;
+use crate::{PlutusDataList, RedeemerList};
+
+use super::{ExUnits, PlutusV1Script, PlutusV2Script};
+
+#[derive(Clone, Debug)]
+#[wasm_bindgen]
+pub struct ConstrPlutusData(cml_chain::plutus::ConstrPlutusData);
```

Comment:
would be good to have a comment on the purpose of `ConstrPlutusData`, since it wouldn't be clear what's that for the ones who just start working with cardano